### PR TITLE
Fix HTML checker url

### PR DESCRIPTION
**Pull request by CYBAI**
I found https://checker.html5.org/ is 502 now. I think it's the `validator`?

### CODE REVIEW

**Comment by annevk** [comment]
@sideshowbarker this is all you 😊

**Comment by sideshowbarker** [comment]
Yes we should stop referencing checker.html5.org directly. https://validator.w3.org/nu now fully conforms to the actual HTML spec ーwhereas it previously had conflicted with the spec in cases where the (now dead) W3C fork had requirements which conflicted with the spec. So the need for checker.w3.org has now been obviated.

**Comment by CYBAI** [comment]
@sideshowbarker Should I update the link to use https://validator.w3.org/nu instead 👀?

**Comment by sideshowbarker** [comment]
> @sideshowbarker Should I update the link to use https://validator.w3.org/nu instead eyes?

No, I think the way you already did the patch is the right way. That way preserves a useful level of indirection.

**Comment by CYBAI** [comment]
@sideshowbarker I'd like to ask one more question which might be off-topic for this PR 🙇

I found there're a `Checker` and a `Validator` in https://whatwg.org/validator/ .

The `Checker` is same as the one I removed in this PR. Not sure if it should be removed as well?

And for the `Validator`, I found its url is https://html5.validator.nu/ which is different with https://validator.w3.org/nu you provided in https://github.com/whatwg/whatwg.org/pull/247#issuecomment-484323137 . If they're different, which one should we provide in the whatwg validator page?

Thanks!

**Comment by sideshowbarker** [comment]
> And for the `Validator`, I found its url is https://html5.validator.nu/ which is different with https://validator.w3.org/nu you provided in [#247 (comment)](https://github.com/whatwg/whatwg.org/pull/247#issuecomment-484323137) . If they're different, which one should we provide in the whatwg validator page?

At this point, https://validator.w3.org/nu/

https://html5.validator.nu/ is these days often weeks or months out of date with bug fixes and updates to the sources

**Comment by CYBAI** [comment]
> At this point, https://validator.w3.org/nu/
> 
> https://html5.validator.nu/ is these days often weeks or months out of date with bug fixes and updates to the sources

Thanks! Will send another PR to fix it!

**Comment by annevk** [comment]
Maybe update this PR to also account for that?

**Comment by CYBAI** [comment]
> Maybe update this PR to also account for that?

Ok! Will update later 🙆‍♂️

**Comment by CYBAI** [comment]
Because it's in different page, I committed in separate commit. If I need to squash, please let me know! Thanks!

File: whatwg.org/validator/index.html [hunk 1]
```diff
@@ -31,12 +31,8 @@ <h2>HTML Conformance Checkers</h2>
 
 <ul>
  <li>
-  <p><a href="https://html5.validator.nu/">Validator.nu (X)HTML5 Validator</a></p>
-  <form method="get" action="https://html5.validator.nu/"><p><label>Address of page to check: <input type="url" name="doc" pattern="(?:(?:https?://.+)|(?:data:.+))?" title="Absolute URL (http, https or data only) of the document to be checked."></label> <input name="submit" value="Check" type="submit"></p></form>
- </li>
- <li>
-  <p><a href="https://checker.html5.org/">Nu Html Checker</a></p>
-  <form method="get" action="https://checker.html5.org/"><p><label>Address of page to check: <input type="url" name="doc" pattern="(?:(?:https?://.+)|(?:data:.+))?" title="Absolute URL (http, https or data only) of the document to be checked."></label> <input name="submit" value="Check" type="submit"></p></form>
+  <p><a href="https://validator.w3.org/nu/">Validator.nu (X)HTML5 Validator</a></p>
```
Comment (sideshowbarker) [inline whatwg.org/validator/index.html]:
```suggestion
  <p><a href="https://validator.w3.org/nu/">Nu Html Checker</a></p>
```

**Pull request merged**